tests/test_conditional_steps.py

```diff
@@ -95,18 +95,22 @@ def _build_step_command(files_to_create, exit_code) -> List[str]:
         return ["bash", "-c", ";".join(commands)]
 
     def _write_config_file(self, steps_info) -> None:
-        true_branch_config = self._build_branch_configuration_string(steps_info.true_branch_steps)
-        false_branch_config = self._build_branch_configuration_string(steps_info.false_branch_steps)
+        true_branch_config = self._build_configuration_string(steps_info.true_branch_steps)
+        false_branch_config = self._build_configuration_string(steps_info.false_branch_steps)
+        cs_children_config = self._build_configuration_string(steps_info.conditional_step.children)
```

Comment:
"cs" is not a very obvious abbreviation for "conditional step". Maybe replace it with word "conditinal", or "condition"?